lists/global.csv

```diff
@@ -1300,3 +1300,4 @@ https://equal-eyes.org/,LGBT,LGBT,2019-11-18,OutRight International,
 https://arc-international.net/,LGBT,LGBT,2019-11-18,OutRight International,
 https://www.eurobicon.org/,LGBT,LGBT,2019-11-18,OutRight International,
 https://etherscan.io/,COMM,E-commerce,2019-12-04,OONI,blocked in china according to https://www.coindesk.com/chinas-internet-firewall-has-blocked-access-to-ethereum-block-explorer-etherscan-io
+https://dns.google/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HACK,Hacking Tools,2019-12-04,Chelchela,
```

Comment:
Why is this categorised as hacking?

Reply:
I thought it was better for DoH. Which one do you prefer?

HOST = Web hosting services, blogging and other online publishing platforms.
HACK = Sites dedicated to computer security, including news and tools. Includes malicious and non-malicious content.
ANON = Sites that provide tools used for anonymization, circumvention, proxy-services and encryption.

Reply:
Now that I think of it, I see what you mean. 

I suppose that for consistency, we should label it like the other DoH service bellow (which are using `HOST`), but we should perhaps extend the definition of the `HOST` category to also include infrastructural services.
cc @sneft

Reply:
done